Workbooks/Virtual Machines - Network Dependencies/Connections Overview - Single VM/Connections Overview.workbook

```diff
@@ -74,12 +75,61 @@
             ],
             "isHiddenWhenLocked": true,
             "queryType": 0,
-            "resourceType": "microsoft.resources/subscriptions"
+            "resourceType": "microsoft.resources/subscriptions",
+            "value": null
+          },
+          {
+            "id": "b12f03d2-87cb-4a88-b4c1-6e24b28cb831",
+            "version": "KqlParameterItem/1.0",
+            "name": "ConnectionLogs",
+            "type": 1,
+            "query": "VMConnection\r\n| where _ResourceId contains \"{WorkbookResource}\"\r\n| take 10\r\n| count",
```

Comment:
this query is almost exactly the same as the Test query parameter, except this one has a where and takes 10 instead of 1?
can you combine these 2 (expensive) queries into just one that does a union and return something like "not onboarded", "not this resource", "onboarded" ?

Reply:
I removed the Test parameter and consolidated the logic. I also updated the DisclaimerText parameter to use criteria (this was initially created before the criteria feature existed so it was previously using Logs query)